src/commands/mixu.ts

```diff
@@ -49,7 +60,7 @@ class Mixu {
       }
       if (
         i % 4 != 3 && // not the last tile in the row
-        (tile + 1) % 4 != 0 && // not a row end tile
+        tile % 4 != 0 && // not a row end tile
```

Comment:
This will reintroduce the scoring bug in #59

Reply:
Thanks, I'm not sure how that snuck in. There shouldn't be any changes to the scoring in this PR.